back/engines/commercial/custom_style/config/schemas/style.schema.json

```diff
@@ -88,7 +88,7 @@
     },
     "color": {
       "anyOf": [
-        {"title": "hex", "type": "string", "pattern": "^#[0-9a-f]{3}([0-9a-f]{3})?$"},
+        {"title": "hex", "type": "string", "pattern": "^#[0-9a-fA-F]{3}([0-9a-fA-F]{3})?$"},
```

Comment:
Had to do this because it didn't accept changing the overlay banner color change. Wonder how this ever worked 🤔